src/os/ui/draw/draw.js

```diff
@@ -1,8 +1,122 @@
 goog.provide('os.ui.draw');
+goog.provide('os.ui.draw.utils');
```

Comment:
The utils namespace is superfluous. We generally prefer to add utilities/static functions directly onto the root namespace, in this case things like `os.ui.draw.getGridFromFeature` would be fine.